Dockerfile-build

```diff
@@ -1,4 +1,4 @@
-FROM mhart/alpine-node:4.7.2
+FROM mhart/alpine-node:base-4.7.2
```

Comment:
The tag `4.7.2` means nodejs@4.7.2 plus npm. The `base-` prefix means only the node binary, without npm.

The full details can be found in https://hub.docker.com/r/mhart/alpine-node/